File: ASP_Angular_2_SPA - github/ASP_Angular_2_SPA/ClientApp/app/ParsingListViewModel.ts
import { PagingInfo } from './PagingInfo';
import { ParsingInfoSimplified } from './ParsingInfoSimplified';

export class ParsingListViewModel {
    constructor(
        public pagingInfo: PagingInfo = new PagingInfo(),
        public parsingInfoSimplifieds: ParsingInfoSimplified[] = []
    ) { }
}

File: ASP_Angular_2_SPA - github/ASP_Angular_2_SPA/Models/ParsingListViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ASP_Angular_2_SPA.Models
{
    public class ParsingListViewModel
    {       
        public PagingInfo pagingInfo { set; get; }

        public IEnumerable<ParsingInfoSimplified> parsingInfoSimplifieds { set; get; }
    }
}
